### PR TITLE
feat: add organisations list view

### PR DESCRIPTION
**Pull request by maria-j-k**
closing #1078 

https://search-pr-1141.docker-fid.grid.cyf-kr.edu.pl/
https://github.com/cyfronet-fid/eosc-search-service/issues/839

Organisations and Catalogues should be visible only on beta instance.
New environmental variable: SHOW_BETA_COLLECTIONS should be set to True in beta environment and to False in prod environment on backend

### CODE REVIEW

**Comment by Marcelinna** [comment]
I think we should show more authors before we use + number 
image
Maybe something similar to the authors 
image

**Comment by Marcelinna** [comment]
We dont have status closed?
image

**Comment by Marcelinna** [comment]
I don't see the selected filters on the left
image

**Comment by Marcelinna** [comment]
the numbers are not equal
image
image

**Comment by Marcelinna** [comment]
@jarekzet please make this equal
image

**Comment by Marcelinna** [comment]
@maria-j-k 
it is possible show this field ?
image

**Comment by Marcelinna** [comment]
@jarekzet @maria-j-k 
image

**Comment by Marcelinna** [comment]
@jarekzet 
image

File: backend/app/routes/web/search_filters.py [hunk 1]
```diff
@@ -110,3 +115,19 @@ def create_output(res_json: dict) -> dict:
...
+    """
+    Function removing 'status' facet.
+    """
+    facets.pop("project_status", None)
+    return facets
+
+
+def parse_organisation_facets(facets):
+    """
+    Function removing 'status' facet.
```
[the remark this answers is not in the document]
Comment (maria-j-k) [reply backend/app/routes/web/search_filters.py:130]: @wujuu this is "related_resources" field but concerns only Organisations collection

File: ui/apps/ui/src/app/collections/data/all/filters.data.ts [hunk 2]
```diff
@@ -45,14 +56,24 @@ export const allCollectionsFilters: IFiltersConfig = {
...
+      expandArrow: true,
     },
+    // {
+    //   id: 'related_organisation_titles',
+    //   filter: 'related_organisation_titles',
+    //   label: 'Related organisations',
+    //   type: 'multiselect',
+    //   defaultCollapsed: false,
+    //   tooltipText: '',
+    // },
```
[the remark this answers is not in the document]
Comment (maria-j-k) [reply ui/apps/ui/src/app/collections/data/all/filters.data.ts:68]: This filter is not working properly  as multiselect due to the amount of unique values. We were suppose to meet to talk about the issue find a way to make an efficient filter in such cases

**Comment by Marcelinna** [comment]
Sometimes
image

**Comment by Marcelinna** [comment]
@maria-j-k 
When I click Projects / Fundings:210 entities
image
Project tab is not higlighted
image

**Review by Marcelinna — APPROVED** [review]

**Pull request merged**